images/bin/start.sh

```diff
@@ -14,6 +14,10 @@ if [ -d "/var/okteto/cloudbin" ]; then
         fi
 fi
 
+if [ -d "/var/okteto/hooks/start.sh" ]; then
+  /var/okteto/hooks/start.sh
```

Comment:
start.sh might not be executable. I think it's easier to just run `sh /var/okteto/hooks/start.sh` or source i

Reply:
if this command fails, will up fail?

Reply:
I didn't use `sh` to avoid a level of indirection, but I guess it is fine.
About the script failing, I would fail `okteto up`, it would be an unexpected behavior otherwise. If the user wants to ignore errors, she can handle them on their script

Reply:
Cool, let's just document that we expect start to be an executable in that case.

Reply:
@rberrelleza also, we are not showing the output, the user will need to do a `kubectl logs podname`.
I am expecting the hook to be fast, if it does things like installations, I would run it via exec (as we do for the cleanup) so we can provide interactive output

Reply:
I moved the logic to run the script client-side when needed

Reply:
What motivated the change? The log?

Also, is the command running after all the files are synchronized or before all that?

Reply:
Yes, it is to have more control over the logs and errors. It is executed right after the synchronization is finished